The Lawn Mower Picture

```diff
@@ -0,0 +1,52 @@
+<html>
+    <head>
+        <title> HTML & CSS Experimets</title>
+        <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+      
+<div class="container">
+  <h2>Dynamic Tabs</h2>
+  <ul class="nav nav-tabs">
+    <li class="active"><a data-toggle="tab" href="#home">Home</a></li>
+    <li><a data-toggle="tab" href="#menu1">Menu 1</a></li>
+    <li><a data-toggle="tab" href="#menu2">Menu 2</a></li>
+    <li><a data-toggle="tab" href="#menu3">Menu 3</a></li>
+  </ul>
+
+  <div class="tab-content">
+    <div id="home" class="tab-pane fade in active">
+      <h3>HOME</h3>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+    </div>
+    <div id="menu1" class="tab-pane fade">
+      <h3>Menu 1</h3>
+      <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+    </div>
+    <div id="menu2" class="tab-pane fade">
+      <h3>Menu 2</h3>
+      <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
+    </div>
+    <div id="menu3" class="tab-pane fade">
+      <h3>Menu 3</h3>
+      <p>Eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
+    </div>
+  </div>
+</div>
+        <style>
+            body {
+                background-color:rgb(100, 189, 66);
+            }
+        </style>
+    </head>
+    <body>
+        <style="width:1305px;height:780px;padding:10px;border:10px solid #3bb9ff;">
+     <h1  style="color:Cyan;font-family:arial;text-align:center;font-size:300%;">The Lawn Mower Crew</h1> 
+             <P> I wanted the picteure to be centered>
+             <h2>Hot Mowers</h2> 
+              <img src="http://www.likecool.com/Home/Outdoor/F1%20Concept%20Lawn%20Mower/F1-Concept-Lawn-Mower-.jpg" alt="donald"style="width:304px;height:228px; align="middle";>
```

Comment:
You need to add a center to it. It should look like this.
`<center>
<img src="http://www.likecool.com/Home/Outdoor/F1%20Concept%20Lawn%20Mower/F1-Concept-Lawn-Mower-.jpg" alt="donald"style="width:304px;height:228px; align="middle";>
<center>`